specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2023-06-01-preview/mfe.json

```diff
@@ -2286,6 +2287,12 @@
                 }
               ]
             }
+          },
+          {
+            "in": "query",
+            "name": "stage",
+            "description": "Stage for including/excluding (for example) archived entities. Takes priority over listViewType",
+            "type": "string"
```

Comment:
>string [](http://example.com/codeflow?start=21&length=6)

Should it be an enum?

Reply:
It should be string as it is a user-defined value. Also, it was approved as string in the latest GA (2023-04-01) for several APIs so we need it to remain as string to avoid breaking changes and for consistency.